python/apps/taiga/tests/integration/taiga/users/test_api.py

```diff
@@ -285,6 +285,25 @@ async def test_update_my_user_success(client):
     assert response.status_code == status.HTTP_200_OK, response.text
 
 
+#####################################################################
+# DELETE /my/user
+#####################################################################
+
+
+async def test_delete_user_204_success(client):
```

Comment:
According to the [latest convention](https://github.com/taigaio/taiga/blob/main/python/docs/conventions/test_api.convetions.md), it should be `test_delete_user_204_no_content`
After the 204 it goes the http status message, at least a representative word. After that you can add more suffixes if you need it.

Reply:
Done!